packages/tooltip/src/Tooltip.tsx

```diff
@@ -168,6 +168,12 @@ function Tooltip({
     triggerEvent: TriggerEvent,
     triggerProps?: any,
   ) => {
+    if (!React.isValidElement(trigger)) {
```

Comment:
I feel like this doesn't get at the problem with meaningful specificity. That being said, it'd be really tough to. Spoke with @DesignerDave and we're cool with just updating the docs and being more explicit in storybook for now. We can go ahead and remove this warning

Reply:
got it, I removed the warning 👍